lms/djangoapps/courseware/grades.py

```diff
@@ -842,7 +842,9 @@ def _calculate_score_for_modules(user_id, course, modules):
         if module.category not in ignore_categories and (module.graded or module.has_score):
             module_score = scores_client.get(locations[index])
             if module_score:
-                module_percentages.append(module_score.correct / module_score.total)
+                correct = module_score.correct or 0
```

Comment:
It seems like an odd case in that a valid module_score would be returned from ScoresClient but it would not have correct and/or total populated.  Is this something we typically expect or maybe it's a weird scenario that we should think about logging?